gst/nnstreamer/elements/gsttensor_if.c

```diff
@@ -345,6 +345,49 @@ gst_tensor_if_set_property_glist (const GValue * value, GList ** prop_list,
   g_strfreev (strv);
 }
 
+/**
+ * @brief Convert GValue to GList for cv option
+ */
+static void
+gst_tensor_if_set_property_cv_option (const GValue * value, GList ** prop_list)
+{
+  gint64 val;
+  gint length, i;
+  const gchar *param = g_value_get_string (value);
+  gchar **strv = g_strsplit_set (param, ",", -1);
+  GValue tmp = G_VALUE_INIT;
+
+  length = g_strv_length (strv);
+
+  if (length > 2) {
+    ml_loge
+        ("Invalid compared value option. It should be in the form of 'IDX_DIM0: ... :INDEX_DIM_LAST,nth-tensor'(A_VALUE) or 'nth-tensor' (TENSOR_AVERAGE_VALUE)");
+    return;
+  }
+
+  g_value_init (&tmp, G_TYPE_STRING);
+  g_value_set_string (&tmp, strv[0]);
+
+  gst_tensor_if_set_property_glist (&tmp, prop_list, ":");
+
+  /* A_VALUE */
+  if (length == 2) {
```

Comment:
Are there any cases where the length is less than '2' ?
If we need to check if less than '2', we can fix it as below :
line 362 =>  if (length != 2)  
line 374 =>  remove if (length == 2) { }

Reply:
If compared-value option is TENSOR_AVERAGE_VALUE, length will be 1.
So I just checked if length is 2 ( option A_VALUE ) to append zero values and n-th tensor info.

Reply:
thank you.